taichi/python/export_ggui.cpp

```diff
@@ -302,6 +302,10 @@ struct PyWindow {
     // todo: support other ggui backends
     if (!lang::vulkan::is_vulkan_api_available()) {
       throw std::runtime_error("Vulkan must be available for GGUI");
+    } else if (ti_arch == Arch::opengl) {
```

Comment:
Is it better to check ti_arch is one of cpu, vulkan and cuda before checking is_vulkan_api_available so that other backends like metal also have proper error messages?

Reply:
Sure, the code is updated now.